stdlib/universe/aggregate_window_test.flux

```diff
@@ -235,69 +168,355 @@ testcase count_with_nulls {
             |> group(columns: ["_measurement", "_field", "t0"])
     got = do_test(every: 10s, fn: count)
 
-    testing.diff(got, want) |> yield()
+    testing.diff(got, want)
 }
-testcase min_with_nulls {
```

Comment:
There are a bunch of pre-existing tests changed here. What is the motivation for that? I would have expected that there isn't any change in behavior except for fixing the numeric overflow issue so the tests shouldn't have to change either, right?

Reply:
I changed the test case to use a time source of `_start` instead of `_stop` because then I could reuse the function for the new tests. This just meant I had to change the `_time` column to the start instead of the stop, but it didn't meaningfully change the tests themselves.